src/status_im/ui/screens/network_settings/network_details/views.cljs

```diff
@@ -42,7 +45,7 @@
          config]]
        ;; TODO(rasom): uncomment edit-button when it will be functional,
        ;; https://github.com/status-im/status-react/issues/2104
-       #_[react/view {:opacity 0.4}
+       #_[react/view {:opacity 0.4}]
```

Comment:
Is this change of uncommenting form on purpose? Why is there still an uncommented form / TODO?